core/Cargo.toml

```diff
@@ -15,7 +15,7 @@ path = "lib.rs"
 
 [dependencies]
 downcast-rs = "1.1.1"
-futures = { version = "0.3", features = [ "thread-pool", "compat" ] }
+futures = { version = "0.3", features = [ "thread-pool", "compat", "io-compat" ] }
```

Comment:
Is this bit intentional? If so where is it used?

Reply:
I was getting a compile error for http_bench.